public/app/plugins/datasource/loki/querySplitting.ts

```diff
@@ -153,6 +157,44 @@ export function runSplitGroupedQueries(datasource: LokiDatasource, requests: Lok
   return response;
 }
 
+function updateLoadingFrame(
+  response: DataQueryResponse,
+  request: DataQueryRequest<LokiQuery>,
+  partition: TimeRange[],
+  requestN: number
+): DataQueryResponse {
+  if (isLogsQuery(request.targets[0].expr) || isLogsVolumeRequest(request)) {
+    return response;
+  }
+  const loadingFrameName = 'loki-splitting-progress';
+  response.data = response.data.filter((frame) => frame.name !== loadingFrameName);
```

Comment:
i recommend the filtering to be not based on the frame-name, but on the `meta.dataTopic` thing.. i think we set that one anyway, and we can just do something like:
```suggestion
  response.data = response.data.filter((frame) => frame.meta?.dataTopic !== DataTopic.Annotations);
```

WDYT?

Reply:
What about other annotations in the results? E.g. in dashboards.